check_pgactivity

```diff
@@ -5551,6 +5558,18 @@ sub check_streaming_delta {
 
     @rs = @{ query_ver( $hosts[0], %queries ) };
 
+    my $index = 0;
```

Comment:
That could work for me, maybe with a better name (like excluded slaves or something).

To be clear, you propose to have #of slaves + #of excluded slave = number of walsenders?

@ioguix What's your opinion on it?